src/java.base/share/native/libjava/System.c

```diff
@@ -354,7 +360,9 @@ Java_java_lang_System_initProperties(JNIEnv *env, jclass cla, jobject props)
         PUTPROP(props, "sun.desktop", sprops->desktop);
     }
 
+#if 0 /* Exclude, unused. OpenJ9 implements it to assert. */
     ret = JVM_InitProperties(env, props);
+#endif /* Exclude, unused. OpenJ9 implements it to assert. */
```

Comment:
This change is not necessary: the OpenJ9 implementation does nothing for Java 11.

Reply:
Removed it.